homeassistant/components/ipp/__init__.py

```diff
@@ -124,7 +127,7 @@ async def _async_update_data(self) -> IPPPrinter:
             raise UpdateFailed(f"Invalid response from API: {error}") from error
 
 
-class IPPEntity(Entity):
+class IPPEntity(CoordinaorEntity):
```

Comment:
To fix the unused import error:

```suggestion
class IPPEntity(CoordinatorEntity):
```